cli/src/pcluster/templates/cluster_stack.py

```diff
@@ -1229,21 +1268,14 @@ def _add_outputs(self):
 
         CfnOutput(
             self,
-            "HeadNodeInstanceID",
-            description="ID of the head node instance",
-            value=self.head_node_instance.ref,
+            "HeadNodeAutoScalingGroupID",
```

Comment:
Where was the "HeadNodeInstanceID" used? Are we sure that can be replaced by "HeadNodeAutoScalingGroupID"?

Reply:
we can't get instance id from ASG since we use launch template, I don't find this cfn output is used in other place.